extensions/ffmpeg/src/main/java/com/google/android/exoplayer2/ext/ffmpeg/FfmpegDecoder.java

```diff
@@ -87,7 +87,11 @@ public FfmpegDecoderException decode(DecoderInputBuffer inputBuffer,
     }
     if (!hasOutputFormat) {
       channelCount = ffmpegGetChannelCount(nativeContext);
-      sampleRate = ffmpegGetSampleRate(nativeContext);
+      if ("alac".equals(codecName)) {
```

Comment:
Right. Is it a bug that it's not set though, or is is actually intentional that it's not set? I'd like to see FFMPEG get patched if it's a bug. It's fine to also leave this here as a workaround, but it would be good to be able to ref an FFMPEG issue in a comment to explain why this is here.